pkg/mountutil/volumestore/volumestore.go

```diff
@@ -105,19 +123,63 @@ func (vs *volumeStore) Unlock() error {
 	return nil
 }
 
+func (vs *volumeStore) legacyEnsureLink(name string) error {
```

Comment:
If we have a legacy volume, calling this method will add the name inside `volFilePath` and create a symlink from `legacyVolPath` (with name) to `volPath` (with digest).